ifacemonitor/ifacemonitor_test.go

```diff
@@ -405,7 +414,13 @@ var _ = Describe("ifacemonitor", func() {
 				regexp.MustCompile("dummy"),
 			},
 		}
-		im = ifacemonitor.NewWithStubs(config, nl, resyncC)
+		fatalErrC = make(chan struct{})
+		fatalErrCallback := func(err error) {
+			log.WithError(err).Info("Fatal error reported")
+			close(fatalErrC)
+			panic(errFatal)
```

Comment:
Why do we signal via channel and also by panicking here?

Reply:
The channel signals to the test code.  The panic breaks out of the goroutine (and we handle the panic later)